src/button/cta.scss

```diff
@@ -2,33 +2,41 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 @import './button';
+@import './disabled';
 
 @mixin iui-button-cta {
-  @include themed {
-    background-color: t(iui-color-background-positive);
-    border-color: t(iui-color-background-positive);
-    color: t(iui-color-foreground-accessory);
-  }
-
-  > .iui-icon {
+  &,
+  &:focus:not(:focus-visible) {
```

Comment:
focus-visible should NOT appear alongside any other selectors. If there is a comma in your selector, dont use focus-visible

And why are you putting it around the base styles anyway?